src/content/add-to-app/ios/add-flutter-screen.md

```diff
@@ -409,44 +392,36 @@ struct MyApp: App {
 }
 ```
 
-Then, in your view, the `AppDelegate` is accessible as an `EnvironmentObject`.
+Then, in your view, the `AppDelegate` is accessible via environment.
```

Comment:
```suggestion
In your view, access the `AppDelegate` through the `environment` property.
```

Reply:
updated "via environment" => "through the view environment", since environment not the name of the property